core/scripts/vrfv2/testnet/util.go

```diff
@@ -92,6 +110,28 @@ func eoaFundSubscription(e helpers.Environment, coordinator vrf_coordinator_v2.V
 	helpers.ConfirmTXMined(context.Background(), e.Ec, tx, e.ChainID, fmt.Sprintf("sub ID: %d", subID))
 }
 
+func fundOracle(e helpers.Environment, oracleAddress string, fundingAmount int64) {
```

Comment:
Small function signature nit: can we make `fundingAmount` a `*big.Int` instead of `int64`? We might want to fund with a number that doesn't fit in an `int64` (e.g, for a long-running test or something).